modules/contributing.adoc

```diff
@@ -0,0 +1,46 @@
+[id="contributing-to-docs-contributing"]
+= Contribute to Hybrid Cloud Patterns documentation
+:icons:
+:description: Contribute to Hybrid Cloud Patterns documentation
+:imagesdir: ../images
+
+
+== Different ways to contribute
+
+There are a few different ways you can contribute to Hybrid Cloud Patterns documentation:
+
+* Email the Hybrid Cloud Patterns team at `<>@redhat.com`.
+* Create a link:https://github.com/hybrid-cloud-patterns/docs/issues[GitHub] or link:https://issues.redhat.com/projects/MBP/issues[Jira issue] .
+* Submit a pull request (PR). To create a PR, create a local clone of your own fork of the link:https://github.com/hybrid-cloud-patterns/docs[Hybrid Cloud Patterns docs repository], make your changes, and submit a PR. This option is best if you have substantial changes. For more details on creating a PR see <topic_link_to_contribution_workflow>.
+
+== Contribution workflow
+
+When you submit a PR, the [Hybrid Cloud Patterns Docs team] reviews the PR and arranges further reviews by Quality Engineering (QE), subject matter experts (SMEs), and others, as required. If the PR requires changes, updates, or corrections, the reviewers add comments in the PR. When the PR has been reviewed, all updates are complete, and all commits are squashed, the documentation team merges the PR.
+
+
+== Repository organization
+
+//Placeholder to explain how assemblies, modules, images, common/attribute folders are organized.
```

Comment:
Did you want to take care of this in this PR or save it for another PR?

Reply:
Again, a placeholder - at the time of creating the guide we did not have much clarity info. Even as of today, it might still need some work so I'll address it in a separate PR. Commented it for now